server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/get/shard/GetShardSnapshotRequest.java

```diff
@@ -102,4 +103,12 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(repositories, shardId);
     }
+
+    @Override
+    public String getDescription() {
+        final StringBuilder stringBuilder = new StringBuilder("shard").append(shardId).append(", repositories[");
```

Comment:
I think we would need some separator between `shard` and the shard id, so the string looks like `shard=5, ` or `shard[5]` instead of `shard5`.

Reply:
`shardId` is a `ShardId` whose `toString()` already wraps things in brackets, I don't think we need any more delimiters here. See e.g. this test:

https://github.com/elastic/elasticsearch/blob/fb8ba7ac972930c9e3c75faab8325a57d6703fec/server/src/test/java/org/elasticsearch/action/admin/cluster/snapshots/get/shard/GetShardSnapshotRequestSerializationTests.java#L57